File: src/src/gestorAplicacion/servicios/Tiquete.java
package gestorAplicacion.servicios;

import java.io.Serializable;

public class Tiquete implements Serializable {
    private int id;
    private Servicio servicio;

    //Constructor

    public Tiquete(Servicio servicio, int i) {
        this.servicio = servicio;
    }

    //Getters y Setters

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public Servicio getServicio() {
        return servicio;
    }

    public void setServicio(Servicio servicio) {
        this.servicio = servicio;
    }
}

File: Python/Ejecutable/Python/gestorAplicacion/servicios/tiquete.py
from Python.gestorAplicacion.servicios.servicio import Servicio


class Tiquete():
    def __init__(self, servicio: Servicio, id: int):
        self._servicio = servicio
        self._id = id

    #getters y setters
    def getId(self):
        return self._id

    def getServicio(self):
        return self._servicio

    def setId(self, id: int):
        self._id = id

    def setServicio(self, servicio: Servicio):
        self._servicio = servicio